LayoutTests/fast/dom/normalize-doesnt-check-string-length.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+    <script>
+        function f1() {
+            const MAX_LENGTH = 1 << 28;
+
+            const largeString = 'A'.repeat(MAX_LENGTH);
+
+            const container = document.getElementById('container');
+
+            for (let i = 0; i < 9; ++i)
+                container.appendChild(document.createTextNode(largeString));
+
+            container.normalize();
```

Comment:
We should include `../../resources/js-test.js` and use `shouldThrowErrorName` helper here like this:
```js
shouldThrowErrorName('container.normalize()', 'InvalidModificationError');
```